src/main/java/com/mael/payroll/transactions/ChangeEmployeeAddress.java

```diff
@@ -0,0 +1,18 @@
+package com.mael.payroll.transactions;
+
+import com.mael.payroll.Employee;
+import com.mael.payroll.PayrollDBFacade;
+
+public class ChangeEmployeeAddress extends ChangeEmployee {
+
+    private String address;
+
+    public ChangeEmployeeAddress(PayrollDBFacade payrollDB, int employeeId, String address) {
+        super(payrollDB, employeeId);
+        this.address = address;
+    }
+
+    public void change(Employee employee) {
```

Comment:
Do you use Cmd+O to implement missing methods, Option+Enter or type the signature of the method you're overriding manually?
The `@Override` annotation technically is not needed, but a nice hint/documentation to make it clear that that's specific behaviour defined in the subclass.